frameworks/Clojure/luminus/hello/src/hello/routes/home.clj

```diff
@@ -62,14 +39,11 @@
    :headers {"Content-Type" "text/plain; charset=utf-8"}
    :body "Hello, World!"})
 
-
 (defroutes home-routes
   (GET "/"                [] "Hello, World!")
   (GET "/plaintext"       [] plaintext)
   (GET "/json"            [] json-serialization)
-  (GET "/db"              [] single-query-test)
+  (GET "/db"              [] (single-query-test))
   (GET "/db/:queries"     [queries] (multiple-query-test queries))
```

Comment:
Are you sure the query url is correct in this section? In the benchmark_config I see `query_url": "/queries/",` but I don't see a `/queries` url mentioned here